### ISSUE TITLE
Current Version of Optifine is Clashing with Fabric's Porting Library.

### ISSUE THREAD

**Issue by SageTheSnail** [Chocohead/OptiFabric#924]
[latest.log](https://github.com/Chocohead/OptiFabric/files/10145503/latest.log)

Unsure what to do here. I've tried just about all of the previous 1.19.2 compatible Optifine versions. I'm going to see if downloading an older build of fabric for 1.19.2 will fix the issue. Otherwise, seems as though Optifabric isn't going to work with an updated modloader.

**Comment by SageTheSnail** [comment]
Yeah, I couldn't get it to work by installing an older version of the fabric modloader. Really confused about this! Will try to see if it works with all of the other mods in my folder removed.

**Comment by SageTheSnail** [comment]
It was an interaction with The Twilight Forest. Just a head's up to whoever sees this.

**Comment by topia** [comment]
> [latest.log](https://github.com/Chocohead/OptiFabric/files/10145503/latest.log)

I can't find optifabric in your latest log, so it's not an optifine/optifabric problem, or your uploaded latest log is not after described failure.

**Comment by SageTheSnail** [comment]
> > [latest.log](https://github.com/Chocohead/OptiFabric/files/10145503/latest.log)
> 
> I can't find optifabric in your latest log, so it's not an optifine/optifabric problem, or your uploaded latest log is not after described failure.

yeah, I think I may have uploaded the wrong log. It actually loaded fine with Optifabric installed, but once I added Optifine, it crashed. Turns out it was interacting with at least the twilight forest, but I think multiple mods may have been clashing with Optifine. Decided I would rather live without a CIT pack than get rid of the main mod in my modpack.
